packages/nova-react-test-utils/src/relay/storybook-nova-decorator-relay.tsx

```diff
@@ -76,11 +91,44 @@ export const getNovaRelayDecorator: (
   return getDecorator(createEnvironment, initializeGenerator);
 };
 
+export type EnvironmentMockResolversContext<
+  TypeMap extends DefaultMockResolvers = DefaultMockResolvers,
+> = {
+  mock: {
```

Comment:
Will it make sense to document here (or elsewhere) what these context methods can be used for? Or will we just rely on examples instead?

Reply:
Looks good 🙂